src/main/java/uk/gov/hmcts/ccd/endpoint/userprofile/UserProfileController.java

```diff
@@ -54,18 +57,24 @@ public UserProfileController(FindAllUserProfilesOperation findAllUserProfilesOpe
     @ApiResponses(value = {
         @ApiResponse(code = 200, message = "Found User Profiles")
     })
-    public List<UserProfile> getUserProfiles(@ApiParam("Jurisdiction ID")
-                                             @RequestParam("jurisdiction")
+    public List<UserProfileLight> getUserProfiles(@ApiParam("Jurisdiction ID") @RequestParam("jurisdiction")
                                                  final Optional<String> jurisdictionOptional,
-                                             @RequestHeader(value = "actionedBy", defaultValue = "<UNKNOWN>")
+                                                  @RequestHeader(value = "actionedBy", defaultValue = "<UNKNOWN>")
                                                  final String actionedBy) {
         Instant start = Instant.now();
-        List<UserProfile> userProfiles = jurisdictionOptional
-            .map(j -> findAllUserProfilesOperation.getAll(j, actionedBy))
-            .orElse(findAllUserProfilesOperation.getAll());
+
+        List<UserProfileLight> allUserProfiles = jurisdictionOptional
+            .map(j -> findAllUserProfilesOperation.getAllLight(j, actionedBy))
+            .orElseGet(findAllUserProfilesOperation::getAllLight);
+
         final Duration between = Duration.between(start, Instant.now());
+        long betweenInMils = between.toMillis();
+        log.info("Found {} entries", allUserProfiles.size());
+        log.info("Time to execute the query: {} milliseconds", betweenInMils);
```

Comment:
do we need this?

Reply:
Practically no, but it can be used to test the time it took to do the db query. Admin Web UI returns a whole view, not only json and the time for rendering the view is a time for the user-profile call plus generating and transferring the payload.